api/api-template.md

```diff
@@ -0,0 +1,57 @@
+<!-- REMOVE ALL COMMENTS AFTER FILLING OUT TEMPLATE-->
+<!-- Add your function name in the headline below with an accompanying anchor name.  The anchor should be the same as the function name-->
+## function_name() [](function_name) 
+
+<!-- 
+Include a description of the function here.  It should succinctly describe what the function does and any important considerations with regards to function usage.  Include a link to the relevant section in Using TimescaleDB if one exists.
+-->
+
```

Comment:
What about functions that have SQL synctax, vs. function arguments?

https://docs.timescale.com/latest/api#continuous_aggregate-create_view